options/locale/locale_en-US.ini

```diff
@@ -2591,6 +2591,7 @@ find_file.no_matching = No matching file found
 error.csv.too_large = Can't render this file because it is too large.
 error.csv.unexpected = Can't render this file because it contains an unexpected character in line %d and column %d.
 error.csv.invalid_field_count = Can't render this file because it has a wrong number of fields in line %d.
+error.broken_git_hook = Git hooks of this repository seem to be broken. Please follow the <a target="_blank" rel="noreferrer" href="https://docs.gitea.com/help/faq#push-hook--webhook-arent-running">documentation</a> to fix them, then push some commits to refresh the status.
```

Comment:
Yup, there is a longstanding problem: non-English translation would NEVER be updated ....

Reply:
-> #29500

Reply:
I think we can leave it to #29500, and follow the current way of other translations for now.

Reply:
How about 452afc20bd6db791c3d7cfb84dae53431b290069.

Why "Please follow the <a target="_blank" rel="noreferrer" href="%s">documentation</a> to fix them" instead of:
- "Please follow the **%s** to fix them": it has missed the word "documentation".
- "Please follow the **%[1]sdocumentation%[2]s** to fix them": it will be very difficult for translators to understand what it is unless they have read the code.